tests/test_api.py

```diff
@@ -1837,6 +1838,16 @@ class MockConfiguration(object):
             def is_autorebuild_enabled(self):
                 return False
 
+            def get_container_module_specs(self):
+                ModuleSpec = namedtuple('ModuleSpec', 'name, stream, version')
```

Comment:
Would prefer that this came from some place in the code rather than being redefined in multiple tests.

Reply:
Hmm thinking about this, probably it can be just real class with a classmethod `.from_text` that will do validation.

```python
class ModuleSpec(object):
    __slots__ = ('name', 'stream', 'version')

    ...init...

    @classmethod 
    def from_text(cls, text):
       .. validation ..
       return cls(*values)
```